src/SFA.DAS.RoatpOversight.Web/Controllers/OversightController.cs

```diff
@@ -79,7 +80,10 @@ public async Task<IActionResult> ConfirmOutcome(ConfirmOutcomePostRequest reques
             var userId = HttpContext.User.UserId();
             var userName = HttpContext.User.UserDisplayName();
 
-            await _outcomeOrchestrator.RecordOutcome(request.ApplicationId, request.OversightStatus, userId, userName, request.InternalComments, request.ExternalComments);
+            var approveGateway = string.IsNullOrEmpty(request.ApproveGateway) ? default(bool?) : request.ApproveGateway == ApprovalStatus.Approve;
```

Comment:
add some coverage for approveGateway and approveModeration

Reply:
will do - may change this anyway, as I don't like it, but need to decide what to do instead